app/services/translations.ts

```diff
@@ -0,0 +1,81 @@
+// Vendor
+import FastBoot from 'ember-cli-fastboot/services/fastboot';
+import fetch from 'fetch';
+import IntlService from 'ember-intl/services/intl';
+import Service, {inject as service} from '@ember/service';
+import Shoebox from 'ember-boilerplate/services/shoebox';
+
+// Config
+import config from 'ember-boilerplate/config/environment';
+
+export type AvailableLocale = 'en-ca';
+
+const pathForLocale: Record<AvailableLocale, string> = {
+  'en-ca': '/assets/translations/en-ca.json'
+};
+
+export default class Translations extends Service {
+  @service('fastboot')
+  fastboot: FastBoot;
+
+  @service('intl')
+  intl: IntlService;
+
+  @service('shoebox')
+  shoebox: Shoebox;
+
+  async loadForLocale(locale: AvailableLocale) {
+    const translations = await this.fetchTranslations(locale);
+
+    this.intl.addTranslations(locale, translations);
+  }
+
+  private async fetchTranslations(locale: AvailableLocale): Promise<object> {
+    if (this.fastboot.isFastBoot) {
+      return await this.fetchTranslationsFromNetwork(locale);
+    }
+
+    const translations = await this.fetchTranslationsFromShoebox(locale);
+
+    if (translations) return translations;
+
+    return this.fetchTranslationsFromNetwork(locale);
+  }
+
+  private async fetchTranslationsFromNetwork(
+    locale: AvailableLocale
+  ): Promise<object> {
+    const translationsURL = pathForLocale[locale];
+
+    const response = await fetch(translationsURL);
+
```

Comment:
Shouldn't we remove empty lines between constants?

Reply:
The second one calls an async function, I prefer to separate it from the simple assignments